### PR TITLE
qvm-ls --raw-data option added

### PR DESCRIPTION
**Pull request by GammaSQ**
The intention is to have an option that returns raw data without formatting or unnecessary data, mainly so it can easily be parsed by other bash-scripts. Therefore, specified VM-ordering should not be changed and all fields are returned in order as well.

Main drawback: I'm using the internal key-strings, so eg. "last backup" has to be passed as "--raw_data last\ backup"

### CODE REVIEW

File: qvm-tools/qvm-ls [hunk 1]
```diff
@@ -24,7 +24,7 @@
 from qubes.qubes import QubesVmCollection
 from qubes.qubes import QubesHost
 from qubes.qubes import QubesException
-from optparse import OptionParser
+from argparse import ArgumentParser
```
Comment (marmarek) [inline qvm-tools/qvm-ls]: For the next time: please make such changes as a separate commit - one for changing library but without any functional change, then actual change. It will make review much easier.

Comment (GammaSQ) [inline qvm-tools/qvm-ls]: The Problem with OptionParser is that it doesn't allow the 'nargs="+"'-option. I wanted to have as many fields specified with one command as required. With OptionParser, 'nargs=1' is the only meaningful option => for n fields I need n commands.

Comment (marmarek) [inline qvm-tools/qvm-ls]: I'm perfectly fine with this change. It's just about having it in separate commit.

**Comment by marmarek** [comment]
Merged with a minor fix (https://github.com/QubesOS/qubes-core-admin/commit/84677fa70bbc716abf223e639e0bf6f7771465ba)

**Comment by marmarek** [comment]
(for the next time) take a look at https://www.qubes-os.org/doc/code-signing/
We're going to enforce this rule soon (will be announced).
